.github/workflows/publish-jar.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and publish JAR
+
+on: push
```

Comment:
Can we restrict the builds to just master branch?

Reply:
Good point, done.